service/src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -176,10 +176,10 @@ public Optional<Workspace> getWorkspaceIfExists(UUID id) {
   @ReadTransaction
   public Workspace getWorkspace(UUID id) {
     return getWorkspaceIfExists(id)
-        .orElseThrow(
-            () ->
-                new WorkspaceNotFoundException(
-                    String.format("Workspace %s not found.", id.toString())));
+            .orElseThrow(
```

Comment:
If you click and drag, you'll see these are spaces. Intellij has a `Continuation indent` preference that's 8 by default:

![image](https://user-images.githubusercontent.com/10929390/166316643-ca7afcad-5019-4f41-aed1-68408497bf33.png)

4 does look better to me, than 8. I changed mine to 4, and I added a step to Terra Onboarding.

Reply:
If you use the Google Style theme instead of Default, you should get that and other settings for free. I didn't have to change my Continuation indent 
![Screen Shot 2022-05-03 at 10 11 36 AM](https://user-images.githubusercontent.com/53479492/166469700-4d3a0962-3278-4475-b59c-55c9ff0c481e.png)
to 4, for example.

Reply:
Here are the instructions from our DEVELOPER.md describing the standard formatting we are using for MC Terra.

### IntelliJ Setup

1. Open the repo normally (File -> Open)
2. In project structure (the folder icon with a little tetromino over it in the upper
   right corner), make sure the project SDK is set to Java 11. If not, IntelliJ should
   detect it on your system in the dropdown, otherwise click "Add JDK..." and navigate to
   the folder from the last step.
3. Set up [google-java-format](https://github.com/google/google-java-format). We use the
   spotless checker to force code to a standard format. Installing the IntelliJ plug-in
   and library makes it easier to get it in the right format from the start.
4. See some optional tips below in the ["Tips"](#tips) section.

Reply:
I have `google-java-format` plugin enabled. Continuation indent was still 8. I'm guessing [default changed to 8 recently](https://intellij-support.jetbrains.com/hc/en-us/community/posts/4403453685778-Why-is-HTML-continuation-indent-default-8-spaces-), maybe that's why you didn't have to change it @ddietterich .

The [Google scheme](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml) does set continuation indent to 4. Should I add Google Schema to the `IntelliJ Setup` instructions? It's unfortunate that `google-java-format` doesn't take care of everything.

(Side note - There was one instance where even though I used `google-java-format` plugin, `spotlessCheck` still failed. That's why I changed Intellij to run `./gradlew spotlessApply`. I'll let you know next time that happens.)